HW_3_updated.py

```diff
@@ -0,0 +1,49 @@
+import re
+
+text = """homEwork:
+
+  tHis iz your homeWork, copy these Text to variable.
+
+
+
+  You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+
+
+  it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE.
+
+
+
+  last iz TO calculate nuMber OF Whitespace characteRS in this Tex. caREFULL, not only Spaces, but ALL whitespaces. I got 87."""
+
+# count the whitespaces
```

Comment:
It would be better to use "docstrings" if you want to describe what function should do. You just need to put text in triple quotes after the function declaration, for example:
```
def count_whitespaces(count_whitespaces):
    """count the whitespaces"""
```
This makes your code better documented, and you can simply use __doc__ to get function description:
```
print(count_whitespaces.__doc__)
```
More details: https://www.programiz.com/python-programming/docstrings